Makefile

```diff
@@ -74,6 +74,15 @@ endif
 
 KERNEL_NAME := $(shell uname -s | tr A-Z a-z)
 
+# When PKG_CACHE_PATH is defined this means we're running in CI. PKG_CACHE_PATH
+# is currently defined in .buildkite/pipeline.yaml. Otherwise fallback to a
+# default in the home directory.
+ifndef PKG_CACHE_PATH
+	GOLANG_BUILD_CACHE ?= ${HOME}/.cache/go-build
+else
+	GOLANG_BUILD_CACHE ?= ${PKG_CACHE_PATH}/go-build
```

Comment:
could set this to `/tmp` explicitly, to make this a little less magic? :D well. :shrug: